build.gradle.kts

```
@@ -0,0 +1,40 @@
+
+plugins {
+    id("com.hivemq.extension")
+}
+
+group = "com.hivemq.extensions"
+description = "HiveMQ Hello World Extension"
+version = "4.4.4"
+
+
+hivemqExtension {
+    name = "HiveMQ Hello World Extension"
+    author = "HiveMQ"
+    priority = 1000
+    startPriority = 1000
+    mainClass = "$group.HelloWorldMain"
+    sdkVersion = "$version"
+}
+
+dependencies {
+    testImplementation ("com.hivemq:hivemq-mqtt-client:1.2.1")
+    testImplementation("org.junit.jupiter:junit-jupiter-engine:5.6.1")
+    testImplementation("org.junit.jupiter:junit-jupiter-api:5.6.1")
+    testImplementation("com.hivemq:hivemq-testcontainer-junit5:1.1.1")
```

```suggestion
    testImplementation("org.junit.jupiter:junit-jupiter-api:5.6.1")
    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.6.1")
    testImplementation("com.hivemq:hivemq-mqtt-client:1.2.1")
    testImplementation("com.hivemq:hivemq-testcontainer-junit5:1.1.1")
```
I prefer grouping dependencies